src/mol-theme/color/volume-value.ts

```diff
@@ -26,30 +29,87 @@ export const VolumeValueColorThemeParams = {
             [ColorNames.white, 1]
         ]
     }, { offsets: true, isEssential: true }),
+    domain: PD.MappedStatic('auto', {
+        custom: PD.Interval([-1, 1], { step: 0.001 }),
+        auto: PD.Group({
+            symmetric: PD.Boolean(false, { description: 'If true the automatic range is determined as [-|max|, |max|].' })
+        })
+    }),
+    isRelative: PD.Boolean(false, { description: 'If true the value is treated as relative to the volume mean and sigma.' }),
+    defaultColor: PD.Color(Color(0xcccccc)),
 };
 export type VolumeValueColorThemeParams = typeof VolumeValueColorThemeParams
 export function getVolumeValueColorThemeParams(ctx: ThemeDataContext) {
     return VolumeValueColorThemeParams; // TODO return copy
 }
 
-export function VolumeValueColorTheme(ctx: ThemeDataContext, props: PD.Values<VolumeValueColorThemeParams>): ColorTheme<VolumeValueColorThemeParams, ColorTypeDirect> {
-    const scale = ColorScale.create({ domain: [0, 1], listOrName: props.colorList.colors });
+export function VolumeValueColorTheme(ctx: ThemeDataContext, props: PD.Values<VolumeValueColorThemeParams>): ColorTheme<VolumeValueColorThemeParams, any> {
+    if (ctx.volume) {
+        const { min, max, mean, sigma } = ctx.volume.grid.stats;
+        const domain: [number, number] = props.domain.name === 'custom' ? props.domain.params : [min, max];
+        const { colorList } = props;
 
-    const colors: Color[] = [];
-    for (let i = 0; i < 256; ++i) {
-        colors[i] = scale.color(i / 255);
-    }
+        if (props.domain.name === 'auto' && props.isRelative) {
+            domain[0] = (domain[0] - mean) / sigma;
+            domain[1] = (domain[1] - mean) / sigma;
+        }
+
+        if (props.domain.name === 'auto' && props.domain.params.symmetric) {
+            const max = Math.max(Math.abs(domain[0]), Math.abs(domain[1]));
+            domain[0] = -max;
+            domain[1] = max;
+        }
+
+        if (ctx.locationKinds?.includes('direct-location')) {
```

Comment:
What is the meaning of a direct location? This location doesn't seem to carry any information, is is still a location? What does the code inside the if-block do?